package.json

```diff
@@ -1,34 +1,50 @@
 {
-  "name": "unmock",
-  "displayName": "unmock-js",
-  "version": "0.0.40",
-  "main": "dist/index.js",
-  "bin": {
-    "unmock": "./dist/cli.js"
-  },
   "license": "MIT",
   "scripts": {
-    "prepare": "tsc --outDir dist",
-    "pretest": "yarn prepare",
-    "test": "jest"
+    "clean": "rm -rf node_modules ./packages/*/dist ./packages/*/tsconfig.tsbuildinfo",
+    "compile": "tsc --build tsconfig.build.json",
+    "compile:clean": "tsc --build tsconfig.build.json --clean",
+    "watch": "tsc --build tsconfig.build.json --watch",
+    "release": "npm run clean && npm install && npm ci && lerna publish --exact --force-publish=unmock --include-merged-tags",
+    "postinstall": "lerna run prepare && npm run compile",
+    "lint": "prettier-check '**/*.{js,ts}'",
+    "lint-fix": "prettier '**/*.{js,ts}' --write",
+    "test": "jest --verbose",
+    "test:clean": "jest --clearCache",
+    "test:watch": "jest --verbose --watchAll",
+    "testonly": "npm test",
+    "test:ci": "npm run coverage -- --ci --maxWorkers=2  --reporters=default --reporters=jest-junit",
+    "coverage": "npm test -- --coverage",
+    "coverage:upload": "codecov"
```

Comment:
This can also be defined in `.circleci` (unless we want to create coverage reports locally?)

Reply:
true - for now I'll leave it as i was using to test with codecov.